src/pages/questionPage.js

```diff
@@ -48,30 +53,63 @@ const nextQuestion = () => {
 //Checks if selected answer is correct
 const checkAnswer = (evt) => {
   const currentQuestion = quizData.questions[quizData.currentQuestionIndex];
-  const value = evt.target.value;
+  const targetValue = evt.target.value;
  
-   if (value === currentQuestion.correct) {
+   if (targetValue === currentQuestion.correct) {
      answerIsCorrect();
    } else {
      answerIsWrong();
+     showCorrectAnswer();
    }
    //if answer is correct
  function answerIsCorrect() {
   score++;
   evt.target.style.backgroundColor = "green";
   alert(`Correct! Your current score is ${score}`);
 }
-
 //if answer is wrong
-function answerIsWrong() {
+  function answerIsWrong() {
   score;
   evt.target.style.backgroundColor = "red";
   alert(`You will get it right next time! Your current score is ${score}`);
-}
+  alert(`The correct answer is ${currentQuestion.correct}`);
   }
+  function showCorrectAnswer() {
+   ANSWERS_LIST_ID.forEach(x => {
+ if (x.value !==  currentQuestion.correct){
+   x.style.backgroundColor="red";
+ } else {
+   x.style.backgroundColor="green";
+ }
+   });
+  }
+}
   
- 
 
+ //saves user's selected answers 
+const saveAnswer = (e) => {
+  const currentQuestion = quizData.questions[quizData.currentQuestionIndex];
+  const value = e.target.value;
+  const answerObj = {
+    'Question': quizData.currentQuestionIndex + 1,
+    'Answer' : value
+  }
+  userAnswers.push(answerObj);
+  console.log(userAnswers);
+
+ //saves answers on window reload
+ localStorage.setItem("userAnswers", JSON.stringify(userAnswers));
+}
 
+//retrieves user's answers on page reload
+window.onload = function() {
+  if (localStorage.getItem('userAnswers')) {
```

Comment:
Minor note: Because we are making the same call twice here, `localStorage.getItem('userAnswers')` we can extract it to a local variable first to make the code tidier and more efficient.

Something like:
`userAnswersRaw = localStorage.getItem('userAnswers');
if userAnswersRaw {
    userAnswers = JSON.parse(userAnswersRaw);
} else {
    userAnswers = []
}`